test/src/sdk-harness/test_projects/exponentiation/mod.rs

```diff
@@ -0,0 +1,78 @@
+use test_helpers::script_runner;
```

Comment:
wouldn't this file be better named `harness.rs`? `mod` seems weird to me.

Reply:
It's `mod.rs` for all the SDK harness tests, so orthogonal to this PR.

Reply:
This is how we do it atm in the stdlib test suite.
then, in `harness.rs` we collect all the mods like
```
// Add test modules here:

mod auth;
mod call_frames;
mod context;
mod exponentiation;
mod hashing;
mod logging;
mod reentrancy;
mod registers;
mod storage;
mod token_ops;
mod tx_fields;

```

Reply:
got it thanks